ironfist/src/cpp/spell/spells.h

```diff
@@ -115,6 +115,25 @@ enum SPELL_CATEGORY
   SPELL_CATEGORY_ALL = 2,
 };
 
+enum STACK_MODIFYING_EFFECT
+{

```

Comment:
Yep, this is a reasonable place to put it. 

Nitpick: Open brace should go on the same line, as per the style guide